### PR TITLE
dependencies count added

### PR DESCRIPTION
**Pull request by sivaavkd**
Mismatch between count for security issues and number of dependencies affected with the security issues

### CODE REVIEW

**Comment by sivaavkd** [comment]
Fix for https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/issues/316

**Comment by sivaavkd** [comment]
> Screenshot 2019-08-30 at 8 55 16 AM
> 
> Changes looks good to me. The only concern is the gap between the "Dependencies effected" and "Highest CVSS" seems to be too close. Is it possible to increase it by bit?
> 
> Another nit is, change "effected" to "Effected" to keep consistent with rest of the cards.

I tried and not able to increase the gap.  I think, each line item in the card is having fixed gap.  Changed the text to "Effected".

**Comment by arajkumar** [comment]
@invincibleJai , any other concerns?

File: src/app/stack/utils/report-summary-utils.ts [hunk 1]
```diff
@@ -99,13 +100,14 @@ export class ReportSummaryUtils {
                         }
                     }
                     securityIssues += currSecurity.length;
+                    dependenciesEffected += 1
```
[the remark this answers is not in the document]
Comment (sivaavkd) [reply src/app/stack/utils/report-summary-utils.ts]: done

**Pull request merged**